src/onboarding/registration/SelectCountry.tsx

```diff
@@ -32,10 +33,17 @@ export default function SelectCountry({ navigation, route }: Props) {
   )
 
   function onSelect(country: LocalizedCountry) {
-    navigation.navigate(Screens.VerificationStartScreen, {
-      selectedCountryCodeAlpha2: country.alpha2,
-      hideOnboardingStep,
-    })
+    if (keylessBackupFlow) {
```

Comment:
I think this component should take an onSelect callback over an optional keylessBackup enum. That way this component doesn't need to know the details on where it should navigate next and can be reused in the future with no additional changes.
Also, this needs tests